docs/content/dev/2.0.md

```diff
@@ -28,3 +28,14 @@ The software provides two special groups which have no explicit users:
 - Run `yarn install` in both projects
 - Start the backend server in watch mode using `yarn start:dev`. The backend is then accessible on port 3000.
 - Start the frontend dev server using `yarn start`. The frontend is now accessible on port 3001. 
+
+## Entity `create` methods
+
+Because we need to have empty constructs for TypeORM to work. The actual constructors of all our entity classes is a `create` method. These methods should adhere to these few guidelines:
```

Comment:
```suggestion
Because we need to have empty constructors in our entity classes for TypeORM to work, the actual constructor is a separate `create` method. These methods should adhere to these guidelines:
```